lib/surface_site_web/live/events.ex

```diff
@@ -234,11 +239,37 @@ defmodule SurfaceSiteWeb.Events do
                 ## Handle event somewhere else
 
                 Using Surface, the event is **always** passed along with the related target, assuming,
-                by default, that the target is the caller component/view. This should cover most of the
-                cases you have to face when working with events. In the rare cases when you need to handle the
-                event somewhere else, you can explicitly pass the target, e.g., `click={{ "click", target: "#target_id" }}`.
+                by default, that the target is the caller component/view.
+
+                This should cover most of the cases you have to face when working with events.
+                In the rare cases when you need to handle the event somewhere else, you can explicitly
```

Comment:
Maybe change "In the rare cases when" to "If"? I very much got the feeling that I was doing something weird when I was passing my event to `:live_view` 😅

Reply:
Done! :)